rrfs-test/CMakeLists.txt

```diff
@@ -2,6 +2,72 @@
 #!    NOAA/NCEP, National Centers for Environmental Prediction  RDASapp        !
 #!-------------------------------------------------------------------------
 
+###########################
+### Begin function list ###
```

Comment:
@SamuelDegelia-NOAA   Could  we put those functions definition part to a separate include file ( I guess there would be more functions to be added in the future) and then, to be "included" into this file?

Reply:
Good idea, I can do that.

Reply:
@TingLei-NOAA How about this? I just put the `dict_funcs.cmake` inside `RDASApp/rrfs-test/` but I'd be fine with it going somewhere else.

Reply:
Thank @SamuelDegelia-NOAA and @TingLei-NOAA for working on this.

If I understand correctly, I think Ting means a file named like "funcs.cmake" instead of "dict_funcs.cmake" which is limited to dict functions only.

Reply:
Good point, `funcs.cmake` is more generic if we need to add more functions later on. I added new commits that renamed the function file.